.github/workflows/gradle.yml

```diff
@@ -10,12 +10,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+    - uses: actions/checkout@v3
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
       with:
-        java-version: 1.8
-    
+        distribution: 'temurin'
```

Comment:
Is that required?

Reply:
Yes.  Declaring the `distribution` is mandatory either the `setup-java@3`.  We also require JDK 1.8 as `downloadManager` still runs on that on CADC infrastructure.